BranchSDK.xcodeproj/project.pbxproj

```diff
@@ -1635,7 +1622,7 @@
 				5F2211352894A34000C5B190 /* BNCCallbackMap.m in Sources */,
 				5F2210C82894A34000C5B190 /* NSString+Branch.m in Sources */,
 				5F2210DF2894A34000C5B190 /* UIViewController+Branch.m in Sources */,
-				5F2210FE2894A34000C5B190 /* BNCFacebookAppLinks.m in Sources */,
+				5F2211082894A34000C5B190 /* BranchLogoutRequest.m in Sources */,
```

Comment:
Wasn't `BranchLogoutRequest` removed?

Reply:
Ah yeah, looks like i messed up my merge conflict fix. Just removed that line too.